### PR TITLE
[fr] il y a rule

### PR DESCRIPTION
**Pull request by Rebecca-auque**
Y a est une expression familière.

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -34110,7 +34110,7 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
-    <rulegroup id="ESPACE_APRES_POINT" name="Vérifiez l'espace après le point.">
+      <rulegroup id="ESPACE_APRES_POINT" name="Vérifiez l'espace après le point.">
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:34113]: How did that space get in here? Now the rulegroup has the same indent as the AP below...

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -35762,6 +35762,88 @@ though, and has since been slightly modified)-->
...
+                  <token>y</token>
+                  <token>a</token>
+              </antipattern>
+              <antipattern>
+                  <token>y</token>
+                  <token>a</token>
+                  <token>-t-il</token>
+              </antipattern>
+              <antipattern>
+                  <token postag="[RNZ].*" postag_regexp="yes"></token>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:35778]: empty string

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 3]
```diff
@@ -35762,6 +35762,88 @@ though, and has since been slightly modified)-->
...
+                  <marker>
+                      <token>y</token>
+                      <token>a<exception scope="next">t</exception></token>
+                  </marker>
+              </pattern>
+              <message>Cette expression est considérée comme familière.</message>
+              <suggestion>il \1 \2</suggestion>
+              <example correction="il y a">Selon lui, <marker>y a</marker> un problème.</example>
+          </rule>
+          <rule>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:35792]: Please make sure the first rule doesn't steal from the second rule. I.e., the first needs an AP for pas/plus etc.

**Review by udomai — CHANGES_REQUESTED** [review]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 3 above)
Comment (Rebecca-auque) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:35792]: I added the AP required and improved the rule.

**Review by udomai — APPROVED** [review]

**Comment by udomai** [comment]
Manually merged: 22a84773c034a17fcd244e88ad1bd09462497dae

**Pull request closed without merging**